reactive/kata.py

```diff
@@ -120,3 +121,15 @@ def publish_config():
         name='kata',
         binary_path='/usr/bin/kata-runtime'
     )
+
+
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    """Set status during series upgrade."""
+    status_set('blocked', 'Series upgrade in progress')
```

Comment:
Seems like this status could easily get overwritten by other handlers. I recommend setting it in an atexit handler or refactoring the charm to use layer-status.